easyDiffractionApp/Gui/Pages/Analysis/SideBarAdvanced.qml

```diff
@@ -83,47 +126,207 @@ EaComponents.SideBarColumn {
                 }
             }
         }
-
     }
 
-    /*
     EaElements.GroupBox {
-        title: qsTr("Plot settings")
+        title: qsTr("Fitting constraints")
+        enabled: ExGlobals.Constants.proxy.experimentLoaded
+        last: true
 
-        Row {
-            spacing: EaStyle.Sizes.fontPixelSize
+        ExComponents.AnalysisConstraints {}
 
-            EaElements.CheckBox {
-                text: qsTr("Show legend")
-                checked: ExGlobals.Variables.showLegend
-                onCheckedChanged: ExGlobals.Variables.showLegend = checked
-            }
+        Column {
+            spacing: EaStyle.Sizes.fontPixelSize * 0.5
 
-            EaElements.CheckBox {
-                text: qsTr("Show measured")
-                checked: ExGlobals.Constants.proxy.showMeasuredSeries
-                onCheckedChanged: ExGlobals.Constants.proxy.showMeasuredSeries = checked
-            }
+            // Type 1 START
+            Column {
+                spacing: EaStyle.Sizes.fontPixelSize * -0.5
 
-            EaElements.CheckBox {
-                text: qsTr("Show difference")
-                checked: ExGlobals.Constants.proxy.showDifferenceChart
-                onCheckedChanged: ExGlobals.Constants.proxy.showDifferenceChart = checked
+                EaElements.Label {
+                    enabled: false
+                    text: qsTr("Type 1")
+                }
+
+                Grid {
+                    columns: 4
+                    columnSpacing: EaStyle.Sizes.fontPixelSize * 0.5
+                    anchors.horizontalCenter: parent.horizontalCenter
+                    verticalItemAlignment: Grid.AlignVCenter
+
+                    EaElements.ComboBox {
+                        id: dependentPar2
+                        width: 359
```

Comment:
The constraint related elements on this sidebar are the only elements in the app which have their width in absolute values. 359, 47, 65 etc look pretty random. Maybe defining the widths based on the fraction of the total bar width, `EaStyle.Sizes.sideBarContentWidth` as it is done with the three filtering controls on the basic tab would look better?